src/database/entities/Guild.ts

```diff
@@ -29,16 +29,16 @@ export class Guild extends DBEntity {
         guild.owner = await User.createOrUpdate(discordGuild.owner.user);
         return guild.save();
     }
-    /// The name of the guild
+
     @Column()
     public name: string;
 
-    @OneToOne((type) => User, (user) => user, { lazy: true })
+    @ManyToOne(type => User, user => user.ownedGuilds)
     @JoinColumn()
```

Comment:
I removed lazy because JoinColumn will load the owner when the guild is loaded. It will never be a promise and the lazy is ignored (iirc).